frontend/src/routes/room/[id]/+page.svelte

```diff
@@ -56,6 +60,8 @@
         }
       } else if (isIncomingPlayerStateMessage(message)) {
         playerState = message.data
+      } else if (isIncomingTypingIndicator(message)) {
+        map = new SvelteMap(Object.entries(message.data) as [string, number][])
```

Comment:
We should set the timeout for removing typing indicators here itself

```suggestion
        const timeout = setTimeout(() => {
          if (typingIndicators.get(message.username)[0] === message.timestamp) {
              typingIndicators.remove(message.username)
        }, 5000)
        typingIndicators.set(message.username, [message.timestamp, timeout])
```

Then we can place both the timestamp and the timeout in the SvelteMap